Resources/keybinds.yml

```diff
@@ -48,7 +48,9 @@ binds:
   key: S
 - function: Walk
   type: State
-  key: Shift
+# ES START
+  key: Space
```

Comment:
should this really be rebound? Most players likely have shift-walk muscle memory, even if I do think this is the better keybind.